From 31dcb016171570946ad438a2f2e499bd76ada84d Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Tue, 8 Nov 2011 17:39:06 +0800
Subject: [PATCH 0580/2463] ENGR00161643-2 MX6Q-lite : enlarge the mtd
 partition of NOR

64k is too small for uboot.
So enlarge the partition to 256K.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabrelite.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabrelite.c b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
index ee94133..6e0a9fa 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabrelite.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
@@ -441,7 +441,7 @@ static struct mtd_partition imx6_sabrelite_spi_nor_partitions[] = {
 	{
 	 .name = "bootloader",
 	 .offset = 0,
-	 .size = 0x00010000,
+	 .size = 0x00040000,
 	},
 	{
 	 .name = "kernel",
-- 
1.7.7.4

